### DISCUSSION TITLE
change pachage name

### DISCUSSION THREAD

**Discussion by hanyelmehy110** [iBotPeaches/Apktool#3729] [General]
how to change pachage name and app name before compile

**Comment by iBotPeaches** [comment]
Do you mind taking a look at these and writing up a guide for us? Its commonly requested.

 * https://techblogs.42gears.com/how-to-change-package-name-of-any-apk-file-using-apktool/
 * https://stackoverflow.com/questions/36807634/changing-the-package-name
 * https://github.com/iBotPeaches/Apktool/issues/2538
 * https://xdaforums.com/t/how-to-guide-mod-change-package-names-of-apks.2760965/
 * https://gabbe.me/posts/rename-and-sign-apks

**Reply by hanyelmehy110** [reply]
thank you for your answer ,i well check and try to make a guide
